docs/docs/components.mdx

```diff
@@ -1743,7 +1743,28 @@ See [starspace paper](https://arxiv.org/abs/1709.03856) for details.
 
 Entity extractors extract entities, such as person names or locations, from the user message.
 
-
+:::note
+  If you use multiple entity extractors, we advise that each extractor targets an exclusive
+  set of entity types. For example, use [Duckling](components.mdx#ducklinghttpextractor) to extract dates and times, and
+  [DIETClassifier](components.mdx#dietclassifier-1) to extract person names. Otherwise, if multiple extractors
+  target the same entity types, it is very likely that entities will be extracted multiple times.
+
+  For example, if you use two or more general purpose extractors like [MitieEntityExtractor](components.mdx#mitieentityextractor),
+  [DIETClassifier](components.mdx#dietclassifier-1), or [CRFEntityExtractor](components.mdx#crfentityextractor),
+  the entity types in your training data will be found and
+  extracted by all of them. If the [slots](domain.mdx#slots) you are filling with your entity types are of type text,
+  then the last extractor in your pipeline will win. If the slot is of type list, then all results
+  will be added to the list, including duplicates.
+
+  Another, less obvious case of duplicate/overlapping extraction can happen even if extractors focus on different
+  entity types. Imagine a food delivery bot and a user message like `I would like to order the Monday special`.
+  Hypothetically, if your time extractor's performance isn't very good, it might extract `Monday` here as a time for the order,
+  and your other extractor might extract `Monday special` as the meal.
+  If you struggle with overlapping entities of this sort, it might make sense to add additional training data
+  to improve your extractor. If that does not suffice, you can add a
+  [custom component](components.mdx#custom-components) that resolves conflicts in entity
+  extraction according to your own logic.
+:::
```

Comment:
To be honest, I don't know how to think about this long note (and partly about the other one below too). If the note is only under the general Entity extractors heading, then people might miss it (if they click through to component-specific sections). The other thing is that the note is very long. I guess I find the second note below much more helpful, and perhaps this general note could just very briefly:
- say what the issue is with using multiple extractors, and then
- point folks to the more involved reading in component-specific sections

But then it's probably worth having a short note in each component-specific section that'd mention the issue (again, very briefly) & what the current component's role in the issue is, and then perhaps point to one big note (listed e.g. under `RegexEntityExtractor`) which would actually explain the workarounds...

Reply:
For now I would keep it as it is. I can't really think of a way which is obviously far superior to this. I have added specific links to the warnings so people won't have to search for the info themselves but will land at these more detailed explanations when they are actually at risk of this issue of duplicate extraction

Reply:
Alright. @twerkmeister do you know if the CI checks check for out-dated links in the code? Strikes me as an important thing if we include a link in a warning (particularly because I'm not aware of cases where we'd include links like this, evn though I'm a fan of the idea).

Reply:
It doesn't check in the code.

Reply:
there are some warnings that use the links. That's why there are all these doc link constants in `rasa/shared/constants.py`. So the main doc links should work whenever the docs url is changed and the code is changed. But the anchor might be more prone to change and it won't be so easily noticable. That's a fair point. hmm